### ISSUE TITLE
GPLv3 license restricts use of the theme

### ISSUE THREAD

**Issue by PaulSonOfLars** [dchucks/agnes-hugo-theme#3]
Hi, 

First of all, thank for your work on this theme - it looks great!

I would like to suggest changing the license to something less restrictive, such as the MIT license. With GPL, any users using this theme will also be forced to open source their entire website, which is likely to hinder any adoption.

Changing the license to MIT would remove any concerns about legal repercussions.

Let me know what you think!

**Comment by PaulSonOfLars** [comment]
Closing as the repo owner is not the owner of this codebase; cruip.com is.

**Comment by dchucks** [comment]
@PaulSonOfLars Hi Paul, this repo is not owned by Cruip, but they did create the original HTML template that they have been giving away for free, which I simply ported to Hugo. I have to maintain the license terms of the creator. I think it makes sense to contribute back any improvement on the theme as it serves the purpose with which they provide the theme free of cost.

**Comment by PaulSonOfLars** [comment]
I would be happy to contribute improvements to Cruip for their hard work. However, as you mentioned, this repo is not owned by Cruip.

Their terms also mention that they do not allow redistributing their templates, or creating "website generators", which I understand as including this repo: https://cruip.com/terms/

**Comment by dchucks** [comment]
@PaulSonOfLars Paul 
- I hope you understand that the Cruip template is a **pure HTML template**, and I converted it to a Hugo theme. The change you suggested can easily be done on HTML but needs programmatic intervention on Hugo. I am not taking away their "hard work", on the contrary I thought of bringing their work to a new set of users who would not have used it otherwise.
- If the template was released on GPL by Cruip (which is included as part of their downloadable artefact) should I need to worry about other terms of Cruip when I am maintaining the same license? I am not sure about that.  
- Am I redistributing their template? No I am not, mine is a derivative of their work. However, its true that I didn't ask their explicit permission to convert to Hugo because it was being provided as a Free template under GPL. But I will ask them and get an approval anyway. **If they do not approve I will delete this repo.**
- I think the terms link you have included is about the fact that they do not want "_an End User to customize the Template_", which is something I cannot control, I ensured that I didn't change anything on their template.
